webapp/components/file_attachment.jsx

```diff
@@ -46,28 +46,22 @@ class FileAttachment extends React.Component {
         if (filename) {
             var fileInfo = this.getFileInfoFromName(filename);
             var type = utils.getFileType(fileInfo.ext);
-
             if (type === 'image') {
                 var self = this; // Need this reference since we use the given "this"
-                $('<img/>').attr('src', fileInfo.path + '_thumb.jpg').on('load', (function loadWrapper(path, name) {
+                $('<img/>').attr({src: fileInfo.path + '_thumb.jpg', height: '400px'}).on('load', (function loadWrapper(path, name) {
                     return function loader() {
                         $(this).remove();
                         if (name in self.refs) {
                             var imgDiv = ReactDOM.findDOMNode(self.refs[name]);
 
-                            $(imgDiv).removeClass('post-image__load');
-                            $(imgDiv).addClass('post-image');
-
-                            var width = this.width || $(this).width();
-                            var height = this.height || $(this).height();
-
-                            if (width < Constants.THUMBNAIL_WIDTH &&
-                                height < Constants.THUMBNAIL_HEIGHT) {
-                                $(imgDiv).addClass('small');
+                            if (self.props.isSingle) {
```

Comment:
The single images do work, I think I got the kinks worked out. But I'm going to make this my final try at getting single images in.